sympy/solvers/tests/test_solveset.py

```diff
@@ -2092,6 +2095,7 @@ def test_linear_coeffs():
     raises(ValueError, lambda:
         linear_coeffs(1/x*(x - 1) + 1/x, x))
     assert linear_coeffs(a*(x + y), x, y) == [a, a, 0]
+    assert linear_coeffs(sp.numbers.Float(1.0), *syms) == [0, 0, 1.0]
```

Comment:
You can just use 1.0 rather than `sp.numbers.Float(1.0)`

Reply:
And `x` and `y` are defined at the top of the file so the only line needed is `assert linear_coeffs(1.0, x, y) == [0, 0, 1.0]`

Reply:
Done the necessary changes.